packages/sdk-react-native/src/tasks/taskStart.ts

```diff
@@ -18,7 +18,7 @@ export default createTask({
         let metroConfigName: string | undefined;
         const { reactNativePackageName, reactNativeMetroConfigName } = ctx.runtime?.runtimeExtraProps || {};
         if (reactNativePackageName) {
-            customCliPath = `${doResolve(reactNativePackageName)}/local-cli/cli.js`;
+            customCliPath = `${doResolve(reactNativePackageName)}/cli.js`;
```

Comment:
Please consider using `@react-native-community/cli` instead, because [`react-native/cli.js` is deprecated](https://github.com/facebook/react-native/blob/v0.76.5/packages/react-native/cli.js#L22).